text-io/src/main/java/org/beryx/textio/TextTerminal.java

```diff
@@ -177,6 +177,20 @@ default void println(String message) {
         println();
     }
 
+    /**
+     * Prints a decorated line separator with a character of choice determined by the String parameter.
+     * the length parameter dictates how many characters the decorator should be.
+     * the decorator does not need to be a single character ie : '*' ... it could also be ie '+++' if desired.
+     */
+    default void separateLineWithDecorator(String s, int length) {
+        String separator = s;
```

Comment:
Start with an empty string, otherwise the decorator will be repeated `length+1` times.
```
String separator = "";
```

Reply:
Your feedback is much appreciated